### ISSUE TITLE
How to check perplexity

### ISSUE THREAD

**Issue by smolendawid** [facebookresearch/fastText#816]
I don't understand how can be FastText evaluated using some validation set. It there any method in FastText to check it?

How many epochs were used for training the public models? I suppose it was chosen using some validation sets, but can't find a word about it...

**Comment by Witiko** [comment]
I have the same question. The Bojanowski et al. (2017) report language model perplexity for fastText languag models in Table 5:

![received_233260717956752](https://user-images.githubusercontent.com/603082/81496302-faf0a700-92b6-11ea-8daf-dc253b58e681.jpeg)

Is there any support for computing the perplexity of a model in the code?
